CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 
 ### Added
 - Rank score model in causatives page
+- Exportable HPO terms
```

Comment:
Well, actually they were before through the CLI. HPO terms exportable through WUI?

Reply:
Yes, of course!